src/docs/styles/custom.css

```diff
@@ -25,6 +25,15 @@ body,
     display: flex;
 }
 
+.step-container {
+    display: flex;
+    padding: 2%;
```

Comment:
Interesting, what are these styles fixing?

Reply:
The step chart, when rendered in the documentation, spilled slightly out of the container. So, this was to bring it back into bounds.

Reply:
I see, maybe the default size is larger than the container?

Could you check if using the responsive container HOC --like in https://github.com/eventbrite/britecharts-react/blob/master/src/charts/bar/Readme.md-- this gets solved?

This way we could avoid including this custom styles.
I would avoid this styles and use the responsive HOC for this,

Reply:
I implemented the fix as you suggested, and it works! Thanks for pointing me in the right direction.